src/dreaditor/main_window.py

```diff
@@ -84,6 +84,7 @@ def addPaintMenuAction(text: str, config_name: str):
         addPaintMenuAction("Logic Paths", "paintLogicPaths")
         addPaintMenuAction("World Graph", "paintWorldGraph")
 
+        self.updateMenuForRomVersion()
```

Comment:
Would it be better to just not add Boss Rush maps to begin with for 1.0.0? This looks kinda weird. It could definitely look better with improved styling, but it would still be better to just not have them here imo.
![image](https://github.com/user-attachments/assets/61b836b0-5bae-4cd0-be94-559ea8781ce8)

Reply:
good call, decided to make only "loadable" packages appear here. also prevents showing unusable options when a romfs is selected.